### PR TITLE
Update dependency doc

### PR DESCRIPTION
**Pull request by zhan9san**
Related #3851

### CODE REVIEW

File: src/molecule/dependency/ansible_galaxy/__init__.py [hunk 1]
```diff
@@ -11,6 +11,17 @@ class AnsibleGalaxy(Base):
     Additional options can be passed to ``ansible-galaxy install`` through the
     options dict.  Any option set in this section will override the defaults.
 
+    The `role-file` and `requirements-file` search path is `<role-name>`
+    directory. The default value for `role-file` is `requirements.yml` and the
```
Comment (ziegenberg) [inline src/molecule/dependency/ansible_galaxy/__init__.py:15]:
```suggestion
    directory. The default value for `role-file` is `requirements.yml`, and the
```

File: src/molecule/dependency/ansible_galaxy/__init__.py [hunk 2]
```diff
@@ -11,6 +11,17 @@ class AnsibleGalaxy(Base):
...
+    default value for `requirements-file` is `collections.yml`.
+
+    1. If they are not defined in `options`, `molecule` will find them from
```
Comment (ziegenberg) [inline src/molecule/dependency/ansible_galaxy/__init__.py:18]:
```suggestion
    1. If they are not defined in `options`, `molecule` will find them from the
```

**Review by ziegenberg — COMMENTED** [review]
Just a small change in wording and punctuation.

**Review by ziegenberg — APPROVED** [review]

File: src/molecule/dependency/ansible_galaxy/__init__.py [hunk 3]
```diff
@@ -11,6 +11,17 @@ class AnsibleGalaxy(Base):
...
     options dict.  Any option set in this section will override the defaults.
 
+    The `role-file` and `requirements-file` search path is `<role-name>`
+    directory. The default value for `role-file` is `requirements.yml`, and the
+    default value for `requirements-file` is `collections.yml`.
+
+    1. If they are not defined in `options`, `molecule` will find them from the
+    `<role-name>` directory, e.g. `<role-name>/requirements.yml` and
+    `<role-name>/collections.yml`
+    2. If they are defined in `options`, `molecule` will find them from
```
Comment (zhan9san) [inline src/molecule/dependency/ansible_galaxy/__init__.py:21]: @ziegenberg 

Thanks for your review.

Do we need add one `the` word here like above?

Comment (ziegenberg) [reply src/molecule/dependency/ansible_galaxy/__init__.py:21]: I don't think so. In the above example, `directory` demanded a `the` in front. I also checked with Grammarly, and it's happy about it. :)

**Pull request merged**